Makefile

```
@@ -153,7 +153,29 @@ make-proto:
 ###############################################################################
 # nix installation: https://nixos.org/download.html
 nix-integration-test: check-network make-proto
-	nix run -f ./. run-integration-tests -c run-integration-tests
+	nix run -f ./default.nix run-integration-tests -c run-integration-tests "pytest -v -m normal"
+
+nix-integration-test-upgrade: check-network make-proto
```

```suggestion
nix-integration-test-upgrade: check-network
```

Now only the `nix-integration-test-grpc` need the `make-proto` dependency, right?